app/models/agents/http_status_agent.rb

```diff
@@ -66,7 +66,8 @@ def check
     def receive(incoming_events)
       incoming_events.each do |event|
         interpolate_with(event) do
-          check_this_url interpolated[:url], header_array(interpolated[:headers_to_save])
+          check_this_url event.payload[:url] || interpolated[:url],
```

Comment:
Maybe, but I'm not sure.  If an incoming event had a `url` key, I'd presume the flow would have been built on the assumption that the HttpStatusAgent would check a received URL, not it would use a received event just as a trigger.  So, falling back to using the value in the options might not be what the user intended.